packages/design-tokens/package.json

```diff
@@ -13,9 +13,7 @@
   "types": "dist/web/tokens.d.ts",
   "scripts": {
     "build": "style-dictionary build",
-    "clean": "style-dictionary clean",
-    "prepare": "npm run clean && npm run build",
```

Comment:
Something about the "automatically build after running `pnpm install`" behavior felt odd, so I removed it 😄